editor/utils/dom.js

```diff
@@ -16,6 +16,10 @@ export function getBlockDOMNode( uid ) {
 	return document.querySelector( '[data-block="' + uid + '"]' );
 }
 
+export function getBlockWrapperDOMNode( uid ) {
```

Comment:
Can we add a JSDoc block here describing the intended behavior, arguments, and return value? That we describe this as expected to return an element with tabIndex might reveal that it could potentially benefit from a more appropriate name: `getBlockTabbableWrapper` ? I don't feel strongly; proposed name is fine too.

Reply:
Ah yes, forgot to do that.